src/main.js

```diff
@@ -442,15 +442,31 @@ window.onload = () => {
                     let res = parseInt(e.target.getAttribute('result'));
                     let item = this.suggestResults[res];
 
-                    if (item.action.endpoint === 'suggest') {
+                    if (!item.action) {
+                        let type = item.properties.place_type[0];
+
```

Comment:
It looks like we repeat the same/similar code below. What are your thoughts about making this a reusable function to select the max zoom level?

Reply:
Done!